pkg/neorpc/events.go

```diff
@@ -5,6 +5,9 @@ import (
 	"errors"
 )
 
+// MaxFeeds is the maximum number of subscriptions per one client.
+const MaxFeeds = 16
```

Comment:
It's an internal server thing, client should have no assumptions over what this value is.

Reply:
Removed.